app/views/partials/account_summary/vat/vat_var/prompt_to_activate_no_new_pin.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import connectors.models._
+@import utils.EmacUrlBuilder
+@import config.FrontendAppConfig
+
+@(emacUrlBuilder: EmacUrlBuilder, vatDecEnrolment: VatDecEnrolment, appConfig: FrontendAppConfig, currentUrl: String,
+  forCard: Boolean)(implicit request: Request[_], messages:Messages)
+
+@if(forCard){
+  <h3 id="change-vat-details-header">@messages("vat.vat_var_for_card.change_vat_details_online")</h3>
+} else {
+  <h2 id="change-vat-details-header">@messages("vat.vat_var_for_card.change_vat_details_online")</h2>
+}
```

Comment:
Should they not be the same? `<h2>` or `<h3>`?

Reply:
An h3 is consistent with the rest of the card design, an h2 with the rest of the subpage design.